product_barcode_generator/views/res_company_view.xml

```diff
@@ -2,21 +2,21 @@
 <openerp>
```

Comment:
Can someone make a script and fix modules (in all repositories) at once, instead of manual fixing each time?